joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/passes/Defines.scala

```diff
@@ -31,11 +35,12 @@ object Defines {
   def getBuiltInType(typeInString: String) = s"${GlobalTypes.kernelPrefix}.$typeInString"
 
   object RubyOperators {
-    val hashInitializer = "<operator>.hashInitializer"
-    val association     = "<operator>.association"
-    val splat           = "<operator>.splat"
-    val regexpMatch     = "=~"
-    val regexpNotMatch  = "!~"
+    val backticks: String = "<operators>.backticks"
```

Comment:
```suggestion
    val backticks: String = "<operator>.backticks"
```

We're already inconsistent with `<operator>` vs. `<operators>` in some places. But we should go with `<operator>` whenever we don't need to worry about compatibility.